lib/s3storage.php

```diff
@@ -42,7 +42,7 @@
 class S3Storage implements IObjectStore, IVersionedObjectStorage {
 
 	/**
-	 * @var S3Client
+	 * @var S3Client|null
```

Comment:
let the analysis tools know that private vars can start off null, otherwise there are complaints about "lazy init" `if` statements. The analysis tools think the `if` statements can never be false.
(applies to a few diff lines in this PR)